.laminas-ci.json

```diff
@@ -5,6 +5,7 @@
         {"name": "PHPUnit on PHP 7.1 with locked dependencies"}
     ],
     "ignore_php_platform_requirements": {
-        "8.2": true
+        "8.2": true,
+        "8.4": true
```

Comment:
probably only 8.4 definition is enough, as 8.3 passed?

Reply:
Hmm...agree